docs/nodejs-runtime/master.adoc

```diff
@@ -79,6 +79,15 @@ include::topics/assembly_cache-mission-booster.adoc[leveloffset=+2]
 :parameter-runtime-anchor!:
 
 
+// MESSAGING WORK QUEUE
+:context: messaging-work-queue
+:parameter-runtime: nodejs
```

Comment:
Look at previous lines in this file. The way you guys set the `parameter-runtime` and `parameter-runtime-anchor` attributes makes me very very nervous! ;-)

Reply:
@Ladicek Agreed,. do not like that one either:) I'll work with @tradej to figure out a way to improve on that.